server/Makefile

```
@@ -876,21 +886,22 @@ vet-api: setup-go-work
 vet-api: export GO := $(GO)
 vet-api: export GOBIN := $(GOBIN)
 vet-api: export ROOT := $(ROOT)
-vet-api: ## Run mattermost go vet to verify api4 documentation, currently not passing
-	cd ../tools/mattermost-govet && $(GO) install .
+vet-api: install-govet ## Run mattermost go vet to verify api4 documentation, currently not passing
```

checkmake (0.3.2)

[warning] Target "vet-api" defined multiple times (lines 885 and 889).

(uniquetargets)